apps/browser/lib/hooks/useEvoService.ts

```diff
@@ -61,6 +61,20 @@ export const useEvoService = (
   // Helpers
   const updateUserFiles = useUpdateUserFiles();
 
+  const setChatLog = (chatLog: ChatLog[]) => {
+    // If the first message is the user's goal, append
+    // an informative message from Evo
+    if (chatLog.length === 1 && chatLog[0].user === "user") {
+      setChatLogState([
```

Comment:
Do we care that this won't be persisted (and visible when user refreshes)?
I'm assuming not, but just checking

Reply:
No this is intended to be client-side only.